package/samplers/value_at_risk/_gp/acqf.py

```diff
@@ -188,39 +188,89 @@ def __init__(
         assert 0 <= confidence_level <= 1
```

Comment:
Not directly related to changes of this PR, but if `confidence_level=1` then the argument `k` to `torch.topk` in `LogCumulativeProbabilityAtRisk.eval_acqf` is `0.0` and it raises an error. How about subtracting a small number if it's 1.0?